src/seedsigner/views/seed_views.py

```diff
@@ -159,11 +159,12 @@ def run(self):
     Loading seeds, passphrases, etc
 ****************************************************************************"""
 class LoadSeedView(View):
-    SEED_QR = ButtonOption("Scan a SeedQR", SeedSignerIconConstants.QRCODE)
+    SEED_QR = ButtonOption(" Scan a SeedQR", SeedSignerIconConstants.QRCODE)
     TYPE_12WORD = ButtonOption("Enter 12-word seed", FontAwesomeIconConstants.KEYBOARD)
     TYPE_24WORD = ButtonOption("Enter 24-word seed", FontAwesomeIconConstants.KEYBOARD)
+    TYPE_SSS = ButtonOption("Recover from SSS", FontAwesomeIconConstants.KEYBOARD)
```

Comment:
Maybe "Shamir recovery" instead? "Shamir rebuild"? "Reassemble Shamir" (too long?)?

Ooh, more maybe just "Shamir shares"?

"SSS" risks being too mysterious, even for a user who has explicitly enabled Shamir support in Settings.

Reply:
Same for the variable name itself.

```suggestion
    TYPE_SHAMIR = ButtonOption("Recover from SSS", FontAwesomeIconConstants.KEYBOARD)
```